### DISCUSSION TITLE
1.1 Release: String Freeze, and testing

### DISCUSSION THREAD

**Discussion by cme** [hydrogen-music/hydrogen#1203] [General]
Hi all!

The last features planned for the 1.1 release have been merged to `master`, including all messages and other translatable items. Translation `*.ts` files have been updated from current sources and are ready for language updates.

We are now in String Freeze, so translation updates can proceed without risk of changes to source strings (except any that might be made necessary by bug fixes or missing translatables).

Development on `master` for 1.1 is now limited to bugfixes. As such it would be useful if as many people as possible give it as much of a test drive as possible so we can find and address issues before proceeding to a Release Candidate.

Thanks!

**Comment by theGreatWhiteShark** [comment]
Also please note that we are **not** asking for translations of the manual (as mentioned in the "X ways to contribute to Hydrogen"). There is a major rewriting of the manual going on and new translations would become obsolete before the release.

**Comment by trebmuh** [comment]
[French translation updated](https://github.com/hydrogen-music/hydrogen/commit/7b5dd5880d71101bc77ccfb57eb86eeb51ab13dc).

As said on the ML, there is only one string left untranslated: "`Sample length to beat:`" since I don't know what it does. 
-> Instrument Rack
---> Layers
-----> Edit Layer

If someone would be nice to explain me that, it'd be very appreciated.

**Reply by trebmuh** [reply]
Replied on the ML by Stefano.
> playing with it seems to me that it stretches the length of the sample to match the length of the selected beat at current bpm (in whole note units) without pitch shift.

So all is good for the French translation for 1.1 now.
